### ISSUE TITLE
The question

### ISSUE THREAD

**Issue by yaolp123** [DasLab/superfolder-covid-mrna-vaccines#7]
Hello, I read your project, but I have a question, is the project could be runned given the amino acid sequence and get the RNA sequence?  Looking forward to your letter.

Best regards.

**Comment by everyday847** [comment]
Yes, the step prior to (1) in "where it comes from" is "decide on an amino acid sequence for which you might like to optimize an RNA sequence."

If the sequence is closely related to an existing covid spike sequence already treated here, the workflow described in (8) is reasonable and prevents a lot of repeated work: if you have a small set of mutations, their effects on optimal RNA sequence may be limited and local and require only codon enumeration for those mutations. But if you have a totally new sequence, you probably need to start from scratch in that workflow at step (1).
